### PR TITLE
Added Hubspot setup guide

### PR DESCRIPTION
**Pull request by AmanGuptAnalytics**
Added setup guide for Hubspot, Please review @adrianbr & @TyDunn .

### CODE REVIEW

File: docs/website/docs/pipelines/Hubspot.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
+---
+title: Hubspot
+description: dlt pipeline for Hubspot API
+keywords: [hubspot api, hubspot pipeline, hubspot]
+---
```
Comment (TyDunn) [inline docs/website/docs/pipelines/Hubspot.md:5]:
```suggestion
---

```

Comment (TyDunn) [reply docs/website/docs/pipelines/Hubspot.md:5]: @rahuljo adding a space here

File: docs/website/docs/pipelines/Hubspot.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+
+# HubSpot
+
+HubSpot is a customer relationship management (CRM) software and inbound marketing platform that helps businesses to attract visitors, engage customers, and close leads. 
+
+The `dlt` HubSpot pipeline allows you to automatically load data from HubSpot onto a [destination](https://dlthub.com/docs/destinations) of your choice. It loads data from the following endpoints:
```
Comment (TyDunn) [inline docs/website/docs/pipelines/Hubspot.md:11]:
```suggestion
The `dlt` HubSpot pipeline allows you to automatically load data from HubSpot into a [destination](https://dlthub.com/docs/destinations) of your choice. It loads data from the following endpoints:
```

File: docs/website/docs/pipelines/Hubspot.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+| Companies | information about organizations  |
+| Deals | deal records, deal tracking |
+| Products | goods, services |
+| Tickets | request for help from customers or users |
+| Quotes | pricing information of a product |
+| Web analytics  | events |
+
+## Get API credentials
+
+```
```
Comment (TyDunn) [inline docs/website/docs/pipelines/Hubspot.md:25]:
```suggestion
:::info
```

Comment (TyDunn) [reply docs/website/docs/pipelines/Hubspot.md:25]: @AmanGuptAnalytics @rahuljo we should use [Admonitions](https://docusaurus.io/docs/markdown-features/admonitions) for things like this

File: docs/website/docs/pipelines/Hubspot.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+| Tickets | request for help from customers or users |
+| Quotes | pricing information of a product |
+| Web analytics  | events |
+
+## Get API credentials
+
+:::info
+**Please note:**
+As of November 30, 2022, HubSpot API Keys are being deprecated and are no longer supported. Instead, it would be best to authenticate using a private app access token or OAuth.
+```
```
Comment (TyDunn) [inline docs/website/docs/pipelines/Hubspot.md:28]:
```suggestion
:::
```

File: docs/website/docs/pipelines/Hubspot.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+| Deals | deal records, deal tracking |
+| Products | goods, services |
+| Tickets | request for help from customers or users |
+| Quotes | pricing information of a product |
+| Web analytics  | events |
+
+## Get API credentials
+
+:::info
+**Please note:**
```
Comment (TyDunn) [inline docs/website/docs/pipelines/Hubspot.md:26]:
```suggestion
```

File: docs/website/docs/pipelines/Hubspot.md [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+| Companies | information about organizations  |
+| Deals | deal records, deal tracking |
+| Products | goods, services |
+| Tickets | request for help from customers or users |
+| Quotes | pricing information of a product |
+| Web analytics  | events |
+
+## Get API credentials
+
+:::info
```
Comment (TyDunn) [inline docs/website/docs/pipelines/Hubspot.md:25]:
```suggestion
:::note
```

File: docs/website/docs/pipelines/Hubspot.md [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+1. Log in to your HubSpot account and go to ⚙️ settings in the main navigation bar.
+2. In the left sidebar menu under **Account Setup** select "Integrations" and then click on "Private Apps".
+4. Click on “Create a private app”.
+5. Fill in the “Basic Info” tab, by specifying a name and description.
+6. Next go to the “Scopes” tab and select the following permissions:
+    1. All the read scopes under the CMS, CRM, and Settings options.
+    2. The following under the Standard options:
+    
+    ```
+
```
Comment (TyDunn) [inline docs/website/docs/pipelines/Hubspot.md:40]:
```suggestion
```

**Review by TyDunn — CHANGES_REQUESTED** [review]
@rahuljo This should only include the Hubspot pipeline. For some reason, the Zendesk files are in here too

**Review by rahuljo — DISMISSED** [review]

**Comment by TyDunn** [comment]
@rahuljo why did you accept? are you fixing this problem?

**Review by TyDunn — APPROVED** [review]
LGTM!!

**Review by TyDunn — APPROVED** [review]

**Pull request merged**